resources/views/update-request.blade.php

```diff
@@ -83,8 +87,32 @@ public function rules()
             @endforeach
     @endif
 @endif
-
+@if($containsPublishedAtColumn)'publish_now' => ['nullable', 'boolean'],
+            'unpublish_now' => ['nullable', 'boolean'],
+@endif
         ];
     }
 @endif
+
+@if($containsPublishedAtColumn)
+    /**
+    * Modify input data
+    *
+    * @return array
+    */
+    public function getSanitized()
+    {
+        $sanitized = $this->validated();
+
+        if (isset($sanitized['publish_now']) && $sanitized['publish_now']==true) {
```

Comment:
In my opinion, whole method should be present always, end if should be only in this part, then you do not need to have if in controller.